### DISCUSSION TITLE
ImageMagick next and former does not work

### DISCUSSION THREAD

**Discussion by KlemensBarfus** [ImageMagick/ImageMagick#3788] [Help]
Hi !
Displaying images (sometimes resized) with ImageMagick 6.9.7-4 Q16 x86_64 on Ubuntu "next" and "former" does not work. I would expect, that pressing the button or using e.g. space, I would see e.g. the next image in the folder. But instead the display is closed. 
What is the problem ?
Thanks in advance !

**Comment by urban-warrior** [comment]
We recommend upgrading your release of ImageMagick.  Recent versions of ImageMagick do not exhibit the bug you posted.
